backend/api_handler.py

```diff
@@ -114,9 +117,15 @@ def handler(event, context):
         }
 
     if 'authorizer' in event['requestContext']:
-        username = event['requestContext']['authorizer']['claims']['email']
+        if 'claims' not in event['requestContext']['authorizer']:
+            claims = event['requestContext']['authorizer']
```

Comment:
Why this change?

Reply:
Saving `claims` to later reference `claims['auth_time']` later if reauth required